index.html

```diff
@@ -51,120 +52,158 @@ <h2>Let's Connect</h2>
                 </ul>
             </div>
         </section>
-        <div id="portfolios"></div>
-        <section class="container">
-            <div class="card">
-                <div id="first">
-                    <img src="images/61.png" alt="">
-                </div>
-                <div>
-            
-                <h2 class="title">Tonic</h2>
-            
-                    <ul class="field">
-                        <li class="cano">CANOPY</li>
-                        <li class="tag">&#8226; Back End Dev </li>
-                        <li class="tag">&#8226; 2015</li>
-                    </ul>
-                    <p class="instruction">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                    </p>
-                <p class="desk-praa"> A daily selection of privately personalized reads; 
-                    no accounts or sign-ups required.</p>
-           
-                <ul class="trainning">
-                    <li class="other-skills">html</li>
-                    <li class="other-skills">css</li>
-                    <li class="other-skills">javascript</li>
-                </ul>
-                <button class="card-btn">See Project</button>
-            </div>
+<            <div id="portfolios"></div>
+</section>
+
+<section class="container">
+    <div class="pro-container"></div>
+    
+     <!--<div class="card" id="01">
+        <div id="first">
+            <img src="images/61.png" alt="">
         </div>
```

Comment:
- [ ] Please do not push commented out code to your code base, It make your project look messy. Therefore I suggest you remove all commented out code from your project.